actionview/CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+*   `ActionView::Helpers::TranslationHelper#translate` should return nil when passed `default: nil`, as `I18n#translate` does
+
+    ```ruby
```

Comment:
Would you mind putting this example in the PR description or in the commit message? I think it's good that you include the example in the change, but we tend to keep the CHANGELOG compact.